hw/ip/dcd/dcd.core

```diff
@@ -0,0 +1,62 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:dcd:1.0"
+description: "dcd"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:constants:top_pkg
+      - lowrisc:prim:all
+      - lowrisc:ip:tlul
+      - lowrisc:systems:ast_wrapper_pkg
```

Comment:
ah `ast_wrapper_pkg` no longer exists.  You might want to just switch this to `ast_pkg`

Reply:
* a69abe02b N - (origin/rbox_work, rbox_work) [rbox] added the design files (21 hours ago) <Eric Shiu>
* 2aad944f4 N - [rbox] created the actual CSR files from hjson (21 hours ago) <Eric Shiu>
* b5f63ebba N - [rbox] Created the rbox register definition files (21 hours ago) <Eric Shiu>
| * b5a665ff6 N - (HEAD -> dcd_work, origin/dcd_work) [dcd] add design files (22 hours ago) <Eric Shiu>
| * 164be7a8b N - [dcd] Added to hw/Makefile (22 hours ago) <Eric Shiu>
| * 0f733b60f N - [dcd] Created the actual CSR files from hjson (22 hours ago) <Eric Shiu>
| * 08a677832 N - [dcd] Created the register definition file (22 hours ago) <Eric Shiu>
|/  
| * 1f7eaa1b6 N - (new_work) Verible cleanup (22 hours ago) <Eric Shiu>
| * 43438c7a7 N - cleaned up the Lint errors for Silver integration (3 weeks ago) <Eric Shiu>
| * 68f82b243 N - checked in for Silver integration (3 weeks ago) <Eric Shiu>
| * 43ba91df8 N - cleaned the rebase issue with the new_work branch (3 weeks ago) <Eric Shiu>
| * 0519ddcb0 N - cleaned the rebase issue with the new_work files (3 weeks ago) <Eric Shiu>
| * 24e899f4f N - corrected the syntax error (3 weeks ago) <Eric Shiu>
| * 2f7b61606 N - added dcd and rbox lint config (3 weeks ago) <Eric Shiu>
| * 3ff1b1536 N - checked in DCD and RBOX RTLs for Silver (3 weeks ago) <Eric Shiu>
| * 25e98003a N - Seed the DCD files in the master branch for Silver milestone work (3 weeks ago) <Eric Shiu>
| * 65f13a605 N - update rbox RTL files for the silver milestone (3 weeks ago) <Eric Shiu>
| * 66ae4d42f N - initial rbox checkin for Bronze integration(IO list and register definition) (3 weeks ago) <Eric Shiu>
|/  
* f928877c4 N - (upstream/master, origin/master, origin/HEAD, master) [prim] Wire up full_o sync fifo output port in prim_sram_arbiter (3 weeks ago) <Rupert Swarbrick>

My local and remote master branch are up-to-date. I will reference the ast_pkg.